### ISSUE TITLE
[Question] Dragging multiple polygons in canvas mode?

### ISSUE THREAD

**Issue by AndrejGajdos** [w8r/Leaflet.Path.Drag#48]
I am just wondering. Does dragging multiple polygons work in canvas mode? During dragging only one polygon is visible.

I am adding a [link](https://stackoverflow.com/questions/66443135/dragging-multiple-polygons-on-canvas-rerender-in-react-leaflet/66462390#66462390) to so for reference.

**Comment by w8r** [comment]
Can you put them into a `MultiPolygon`?

**Comment by AndrejGajdos** [comment]
@w8r yes I can. Thank you, this is a great idea. I will try it out and let you know.

**Comment by AndrejGajdos** [comment]
@w8r unfortunately, multipolygon doesn't work because it makes holes for overlapping polygons [codesandbox.io](https://codesandbox.io/s/admiring-shape-flmql)

**Comment by w8r** [comment]
There's a `fllRule` option on the `L.Path` styles for that, what makes it alternate like that is the `evenodd` option, you may try some other one from the list [here](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/fill-rule)

**Comment by AndrejGajdos** [comment]
@w8r default value is `evenodd`, if I try `nonzero` it doesn't make any difference [codesandbox.io](https://codesandbox.io/s/nostalgic-cori-871zx)

**Comment by AndrejGajdos** [comment]
Would be great if this plugin supports dragging whole [FeatureGroup](https://leafletjs.com/reference-1.7.1.html#featuregroup). You can consider this as a feature request.

**Comment by w8r** [comment]
Good point!
